Source/WebCore/Modules/modern-media-controls/media/media-controller.js

```diff
@@ -23,6 +23,8 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+const maxNonLiveDuration = 604800;
```

Comment:
It would be more clear to express this like:
```
60 * 60 * 24 * 7
```
That makes it a little more obvious that the duration is 7 days (you could also add a comment).